### ISSUE TITLE
Write tests for unsubmit feature and autogradingResults

### ISSUE THREAD

**Issue by geshuming** [source-academy/cadet#392]
Will need to do this someday

**Comment by jiachen247** [comment]
@geshuming  unsubmit got test alr meh no? I remember writing it!

**Comment by geshuming** [comment]
Yep I saw it while writing tests for autograding results. I don't think this can be covered because an unsubmitted submission will never become visible to the grader though... https://coveralls.io/builds/23995039/source?filename=lib/cadet_web/views/view_helpers.ex#L13
